sdk/package/lib/mainFn/Mounts.ts

```diff
@@ -45,8 +49,11 @@ export class Mounts<Manifest extends T.SDKManifest> {
   }
 
   addAssets(
+    /** @TODO Bonez. I thought the assets array in the manifest was an array of path to asset directories. They have IDs? */
```

Comment:
yes, they have ids. the path to the asset directory uses the id as the folder name when we build the s9pk, so they are kind of the same